config/jobs/kubernetes/sig-testing/kubetest2-canaries.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes/test-infra:
+  - name: kubetest2-gce-deployer-build-up-down
+    always_run: false
+    decorate: true
+    # TODO: uncomment when job has been tested successfully
+    # run_if_changed: "kubetest2/kubetest2-gce/.*"
+    labels:
+      preset-service-account: "true"
+    extra_refs:
+    - base_ref: master
+      org: kubernetes
+      repo: cloud-provider-gcp
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200623-2424179-master
+        command:
+        - /bin/bash
```

Comment:
you'll want the entrypoint to be `runner.sh` it's baked into the image and will activate the service account for you, unless you plan to do that yourself in the test script.

https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/runner.sh

Reply:
Done.